decompiler/analysis/variable_naming.cpp

```diff
@@ -1068,21 +1068,21 @@ std::optional<VariableNames> run_variable_renaming(const Function& function,
       debug_in += fmt::format(" out: {}\n\n", reg_to_string(block_info.output));
     }
 
-    fmt::print("Debug Input\n{}\n----------------------------------\n", debug_in);
+    lg::debug("Debug Input\n{}\n----------------------------------\n", debug_in);
   }
 
   // Create and convert to SSA
   auto ssa = make_rc_ssa(function, rui, ops);
 
   if (debug_prints) {
-    fmt::print("Basic SSA\n{}\n------------------------------------\n", ssa.print());
+    lg::debug("Basic SSA\n{}\n------------------------------------\n", ssa.print());
   }
 
   // eliminate PHIs that are not needed, still keeping us in SSA.
   while (ssa.simplify()) {
   }
   if (debug_prints) {
-    fmt::print("Simplified SSA\n{}-------------------------------\n", ssa.print());
+    lg::debug("Simplified SSA\n{}-------------------------------\n", ssa.print());
```

Comment:
I think this file should stay as prints - the alignment out the output will probably get messed up because the print functions here insert their own `\n`'s